doc/build/changelog/unreleased_20/9752.rst

```diff
@@ -0,0 +1,18 @@
+.. change::
+    :tags: usecase, sql
+    :tickets: 9752
+
+
+    Generalized the MSSQL :func:`.try_cast` function so that other
+    extension dialects can use it (MSSQL is the only builtin dialect
+    that currently supports it).
+
+    This implements a cast where
```

Comment:
**mike bayer** (zzzeek) wrote:

I can fix his later, but slight nit: word wrap the paragraph evenly to 79 chars, or I'll just do it later..

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4605